Increased font size and weight in topbar navigation

nbhankes opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
This is my first pull request here, so please coach me so that I'm doing this according to the VWC standard.


## Description
This is in response to issue #189, though it does not close the issue completely. I'm just starting with the topbar navigation.

I increased the font size and weights of the topbar navigation elements. Increasing the font size changed the responsive behavior of the entire topbar navigation container. So I ended up changing the container to a centered flexbox between the widths of 1300px and 767px in order to replicate current topbar navigation behavior. The centered flex box applies to a new class name I added to the topbar navigation container. The topbar navigation container was previously a Bootstrap ".container-fluid" class, which also applied to other elements. By adding the new class name, I was able to change the topbar navigation container without affecting the other page element with the "container-fluid" class.


## Related Issue
<!--- This project only accepts pull requests related to open issues -->
This is in response to issue #189, though it does not close the issue completely. I'm just starting with the topbar navigation.

Please link to the issue here: 
[Issue 189](https://github.com/Vets-Who-Code/vets-who-code-app/issues/189)

## Motivation and Context
This change increases accessibility for people with vision issues. The increased font size and weight makes the navigation options easier to read.

## How Has This Been Tested?
I brought the project onto my local system and got it running on the browser. When I made the changes, I visually tested them in Chrome dev tools responsive layout tool. I made sure the changes only impacted the intended elements, and verified that the responsive layout functioned the exact same way as the current website does. I did this on all pages.

## Screenshots (if appropriate):
The following link leads to a video comparing the current site to the proposed changes included in this PR: [Youtube: VWC Testing](https://youtu.be/1HFMfaVOUYU)

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project.
   - I'm not sure. 
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
   - No. I don't know much about testing. The only testing I did was visually done in Chrome Dev tools
- [ ] All new and existing tests passed.
   - Not sure. GitHub says I'm able to merge changes to the master branch.

gixxerblade reviewed (changes requested):
-Your current changes are behind the master branch. Make sure you `git checkout master`, `git pull`, `yarn`, then switch to your branch and perform `git merge master`, `yarn`, then you can `yarn develop`.
-Nav css files are in `src/components/Nav/nav.css`. Put your nav changes there
-Make sure your changes do not conflict with the current media query at L77 (which is flexed). https://github.com/Vets-Who-Code/vets-who-code-app/blob/master/src/components/Nav/nav.css#L77

nbhankes commented:
I made CSS changes to account for the new Board link changing the width of the links, got the branch current with master, and updated the test snapshot. If there was a CSS selector for a navigation element within the main.css file, I added my changes to it right there in the main.css file (for instance, changing font size and weight). After adjusting the flex-box @media width in the nav.css file, I was able to delete the flexbox and class name I created in the main.css.

While I was working on the nav bar, I adjusted the padding on the dropdown menu. This way all the links appear as soon as someone hits the hamburger button, instead of having to scroll down. I believe this helps with accessibility because it was not immediately obvious that there were additional links in the drop down menu. See below:

The current drop down menu requires you to scroll to see the donate button:
<img width="718" alt="Currrent dropdown" src="https://user-images.githubusercontent.com/59675806/94439163-e5402900-0154-11eb-811c-ec61fc15ff33.png">

Proposed change included in the PR:
<img width="716" alt="Proposed change" src="https://user-images.githubusercontent.com/59675806/94439158-e3766580-0154-11eb-9ac9-4595d8b0f692.png">
